content/3. Procedurele informatie/4. Techniek/Workshop Git/2. Git Repositories.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 2. Git Repositories
+taxonomie: None
+tags: 
+draft: true
+---
+
+> [!warning] Stuk overslaan?
+> Al een beetje bekend met Git? Dan kun je het eerste deel van deze workshop overslaan en doorgaan vanaf [[3. Git Forks & Branches]]
+
+## Opdracht 1: Repository aanmaken
+>Een repository is de opslagplaats van je project. Hierin staan alle bestanden en de volledige geschiedenis van hun wijzigingen.
+
+**Stap 1:** Log in op GitHub en ga naar de homepagina of je dashboard.
+
+**Stap 2:** Klik op de groene ‘New’ knop aan de linkerkant van de pagina.
+
+> [!info] Tip:
+> Houd de naam van je repository simpel en beschrijvend. Vermijd spaties en speciale tekens (behalve -).
+
+**Stap 2.1:** Geef de repository een naam (bijvoorbeeld FirstRepository) en een beschrijving. Kies ook of je de repository public of private wilt maken.
+
+**Stap 2.2:** Klik op de Create Repository knop.
+
+>[!success] Goed bezig!
+>Je hebt nu je eerste repository aangemaakt. Gefeliciteerd!
+
+  
+## Opdracht 2: Repository clonen
```

Comment:
Misschien een paar (kleine) screenshots toevoegen. Kijk anders even hoe Teun het heeft gedaan met de nummers in de afbeeldingen